File: python/delta.py
def delta(a, b, c): 
    delta =  b**2 - 4*a*c
    
    if delta > 0: 
        return 2

    if delta == 0: 
        return 1

    if delta < 0: 
        return 0

print(delta(0.3, 0.3, 0.4))

File: js/delta.js
function delta(a, b, c) {
    var delta = b ** 2 - 4 * a * c
   
    switch (true) {
        case (delta > 0): 
            console.log('2');
            break;
        case (delta == 0): 
            console.log("1");
            break;
        case (delta < 0): 
            console.log("0");
            break;
    }

}
console.log(delta(0.3, 0.3, 0.4))